examples/hf_llama_conversion/convert_llama_weights_from_hf.py

```diff
@@ -0,0 +1,191 @@
+import json
+import os
+from typing import List, Union
+
+import click
+import torch
+from tqdm import tqdm
+from transformers import LlamaForCausalLM  # @manual
+
+NUM_SHARDS = {
+    "7B": 1,
+    "7Bf": 1,
```

Comment:
I saw that f stands for fine-tuning, but do we actually need to distinguish between these as the number of shards seem to be the same? Might confuse some users.

Reply:
move into tools directory, removed fine tune options as there is no difference.